Cinema/src/main/java/com/example/Cinema/model/Projections.java

```diff
@@ -1,30 +1,34 @@
 package com.example.Cinema.model;
 
-import com.example.Cinema.validator.*;
+import com.example.Cinema.validator.ValidIdHall;
+import com.example.Cinema.validator.ValidIdMovie;
+import com.example.Cinema.validator.ValidTime;
 
+import javax.validation.Valid;
 import javax.validation.constraints.FutureOrPresent;
 import javax.validation.constraints.NotNull;
 import java.time.LocalDate;
 import java.time.LocalTime;
 
-
+@Valid
 public class Projections {
 
     private Integer id;
-    @NotNull(groups = FirstPriorInfo.class)
```

Comment:
This doesn't seem correct. Validations should have groups. Please double check all your validation constraints and re-add groups.

Reply:
Idea was to put default groups in every annotation. Fixed